tests/relay/test_relay_dispatcher.py

```diff
@@ -144,3 +159,56 @@ async def test_event_3(self, event_builder):
             ]
         )
         await dispatcher.handle(client_session, event)
+
+    @pytest.mark.asyncio
+    async def test_legal_timestamp(self, event_builder):
+        nip_22_config = nips_config.nip_22
+        if not nip_22_config:
+            return
+        current_time = time.time()
```

Comment:
the fun part about not working with a method that uses the actual time it to test it specifically without having the whole dispatcher